Makefile.am

```diff
@@ -144,6 +144,9 @@ scss_targets = $(patsubst $(srcdir)/%.scss,$(builddir)/%.css,$(scss_toplevels))
 .scss.css:
 	$(scss_verbose)$(MKDIR_P) $(@D) && $(SCSS) -a $< $@
 
+.yaml.pot:
```

Comment:
This rule target would be appropriate if it was a general rule to turn a .yaml file into a .pot file, one-to-one. In this case it's not, it should be `data/preset/po/presets.pot:`